terraform/groups/ecs-service/vault.tf

```diff
@@ -0,0 +1,19 @@
+variable "hashicorp_vault_username" {
```

Comment:
Can this file be removed and symlinked, similar to this request on a different repo https://github.com/companieshouse/identity-verification-api/pull/15#discussion_r1388030835

Reply:
@Ngriffiths1 symlinked, Thank you.